hack/tools/copy-service-certs.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Copies the certificate data from the operator-controller namespace to the local machine
+
+set -o errexit
+set -o pipefail
+
+ROOT_DIR=$(readlink -f "$(dirname "${BASH_SOURCE[0]}")/..")
+CERT_DIR="${ROOT_DIR}/certs"
+OLMV1_NAMESPACE="olmv1-system"
+
+rm -rf "${CERT_DIR}"
+mkdir -p "${CERT_DIR}"
+function import_certs() {
+    local secret_name=$1
+    data=$(kubectl get secret "${secret_name}" -n "${OLMV1_NAMESPACE}" -o jsonpath="{.data['tls\.crt']}")
+    if [[ -n "${data}" ]]; then
+      echo "${data}" | base64 --decode > "${CERT_DIR}/${secret_name}-tls.crt"
+    fi
+}
+
+for secret in $(kubectl get secrets -n "${OLMV1_NAMESPACE}" -o jsonpath="{.items[*].metadata.name}"); do
+    import_certs "${secret}"
+done
```

Comment:
What are you doing with this certificate?
In the latest incarnation, you should just be able to grab `ca.crt` from the secret as the root for all certificates used between operator-controller and catalogd. You shouldn't have to grab `tls.crt`, which applies only to that certificate.

Reply:
I just copy the certificates to a folder and start operator-controller with --ca-certs-dir so that I can talk to the catalogd endpoint and debug directly from the IDE

Reply:
The ca can change, right? and if I have the tls cert it should work every time - even if the ca changes?